test/CreatorTokenFactory.t.sol

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {CreatorToken} from "src/CreatorToken.sol";
+import {CTBondingCurve} from "src/CTBondingCurve.sol";
+import {IERC20, ERC20} from "openzeppelin/token/ERC20/ERC20.sol";
+import {CTBondingCurve} from "src/CTBondingCurve.sol";
```

Comment:
Duplicate import. First import on line 6.

Reply:
fixed in 029eeda4404b3c94895a3a574b80d4c216db738a